spanner_orm/api.py

```diff
@@ -39,13 +42,16 @@ def run_read_only(cls, method, *args, **kwargs):
   @staticmethod
   def find(transaction, table_name, columns, keyset):
     """Obtains rows with primary_keys from the given table."""
+    _logger.debug('Find table=%s columns=%s keys=%s',
+                  table_name, columns, keyset.keys)
     stream_results = transaction.read(
         table=table_name, columns=columns, keyset=keyset)
     return list(stream_results)
 
   @staticmethod
   def sql_query(transaction, query, parameters, parameter_types):
     """Runs a read only SQL query."""
+    _logger.debug('%s\n%s\n%s', query, parameters, parameter_types)
```

Comment:
this should indicate that the log is from executing a sql query

Reply:
done